src/webapp/components/DeliveryDialog/InstructionsScreen.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogActions from "@material-ui/core/DialogActions";
+import Button from "@material-ui/core/Button";
+import instrutions from "./instructions.json";
+
+const useStyles = makeStyles(() => ({
+  moreInfoActionsContainer: {
+    justifyContent: "flex-start",
+  },
+  backButton: {
+    backgroundColor: "lightgray",
+  },
+}));
+
+const Subtitle = ({ content }) => (
+  <Typography variant="subtitle1">{content}</Typography>
+);
+
+const Paragraph = ({ content }) => (
+  <Typography variant="body2">
+    <p>{content}</p>
+  </Typography>
+);
+
+const UnorderedList = ({ content: children }) => {
+  const { t: str } = useTranslation();
+
+  return (
+    <ul>
+      {children.map((child) => {
+        const ChildNode = componentTypeMap[child.typename];
+        const opts = {
+          ...child,
+          content: Array.isArray(child.content)
+            ? child.content
+            : str("webapp:deliveryNeeded.dialog.instructions", {
+                defaultValue: child.content,
+              }),
+        };
+
+        return React.createElement(ChildNode, { ...opts });
+      })}
+    </ul>
+  );
+};
+
+const ListItem = ({ content, variant }) => (
+  <Typography variant={variant ?? "body2"}>
+    <li>{content}</li>
+  </Typography>
+);
+
+const componentTypeMap = {
+  subtitle1: Subtitle,
+  p: Paragraph,
+  ul: UnorderedList,
+  li: ListItem,
+};
+
+const Instructions = () => {
```

Comment:
lmk if i over-engineered this 👀

Reply:
not at all! having instructions in a separate JSON file will make it easier for future copy updates. i only worry about doing this alongside the use of i18n might be confusing to other mutual aids that forked from us, so we should make sure to document why we're doing this (i imagine to allow for flexibility in structuring the content).